site/guide/fairings.md

```diff
@@ -96,7 +96,8 @@ events is described below:
     a response fairing can be used to provide a response when the greater
     applications fails to by rewriting **404** responses as desired. As another
     example, response fairings can also be used to inject headers into all
-    outgoing responses.
+    outgoing responses. Beware that the body of a `HEAD` request will be
```

Comment:
This seems very out of context.

If you'd like, we can focus on the code for this PR and I can take care of the documentation changes afterwards.